### PR TITLE
Update ciphersuites

### PR DESCRIPTION
**Pull request by alxdavids**
- Specify OPRF ciphersuites explicitly
- Use SSWU for P384
- Update DST used for h2c

### CODE REVIEW

File: draft-irtf-cfrg-voprf.md [hunk 1]
```diff
@@ -1233,37 +1233,61 @@ relevant components. For reasons that are detailed in {{cryptanalysis}}, we only
...
 - H_1: P521-SHA512-SSWU-RO {{I-D.irtf-cfrg-hash-to-curve}}
-  - label: voprf_h2c
+  - h2c_DST: "VOPRF-P521-SHA512-SSWU-RO-"
 - H_2: HMAC_SHA512 {{RFC2104}}
 - H_3: SHA512
 - H_4: SHA512
 - H_5: HKDF-Expand-SHA512
 
-We remark that the 'label' field is necessary for domain separation of the
+We remark that the 'h2c_DST' field is necessary for domain separation of the
```
Comment (chris-wood) [inline draft-irtf-cfrg-voprf.md]: Should we replace "OPRF" and "VOPRF" in the DST strings with "RFCXXX-OPRF" or "RFCXXX-VOPRF" (resp.)?

File: draft-irtf-cfrg-voprf.md [hunk 2]
```diff
@@ -1233,37 +1233,61 @@ relevant components. For reasons that are detailed in {{cryptanalysis}}, we only
 consider ciphersuites that provide strictly greater than 128 bits of security
 {{NIST}}.
 
-## VOPRF-curve448-HKDF-SHA512-ELL2:
+## OPRF-curve448-HKDF-SHA512-ELL2-RO:
 
 - GG: curve448 {{RFC7748}}
 - H_1: curve448-SHA512-ELL2-RO {{I-D.irtf-cfrg-hash-to-curve}}
-  - label: voprf_h2c
+  - h2c_DST: "OPRF-curve448-SHA512-SSWU-RO-"
```
Comment (chris-wood) [inline draft-irtf-cfrg-voprf.md]:
```suggestion
  - hash-to-curve DST: "OPRF-curve448-SHA512-SSWU-RO-"
```

(here and elsewhere)

File: draft-irtf-cfrg-voprf.md (hunk 1 above)
Comment (alxdavids) [reply draft-irtf-cfrg-voprf.md]: Yeah I think that's a good idea.

**Comment by alxdavids** [comment]
Updated, I'll merge if you're happy.

File: draft-irtf-cfrg-voprf.md [hunk 3]
```diff
@@ -1716,4 +1740,445 @@ consistency.
 
 # Test Vectors {#testvecs}
 
-[[TODO: add when done]]
+We provide test vectors for each of the supported VOPRF ciphersuites below. The
+test vectors are also available at
+https://github.com/alxdavids/voprf-poc/tree/master/test_vectors.
+
+## VOPRF-P384-HKDF-SHA512-SSWU-RO
```
Comment (chris-wood) [inline draft-irtf-cfrg-voprf.md]: We should probably keep the JSON representation elsewhere, i.e., not in the main document. Can we reformat this and drop a pointer to a JSON file, perhaps. stored in this repo?

File: draft-irtf-cfrg-voprf.md [hunk 4]
```diff
@@ -1716,4 +1740,445 @@ consistency.
...
+                "20c39f9a26a0af45a1448e3f2b1e842311c5a824b59350b4fedf0919c7b4
+                1ad8f4fb009029bfbd08fad3234a0cffc855"
+            ]
+        }
+    }
+]
+
+## VOPRF-P521-HKDF-SHA512-SSWU-RO
+
+TODO: P521 test vectors
```
Comment (chris-wood) [inline draft-irtf-cfrg-voprf.md]: Can we remove these, and then make issues for these vectors?

File: draft-irtf-cfrg-voprf.md (hunk 3 above)
Comment (alxdavids) [reply draft-irtf-cfrg-voprf.md]: This change is dealt with in #65, I can merge that branch into this one if you think that would be better?

File: draft-irtf-cfrg-voprf.md (hunk 4 above)
Comment (alxdavids) [reply draft-irtf-cfrg-voprf.md]: Same as above.

File: draft-irtf-cfrg-voprf.md (hunk 3 above)
Comment (chris-wood) [reply draft-irtf-cfrg-voprf.md]: Either works for me! Let’s just merge this and fix after.

**Pull request merged**